src/pages/main/components/selected-dataset/build-flow/components/cards/description-card/description-card.store.ts

```diff
@@ -0,0 +1,93 @@
+import { action, makeAutoObservable, reaction, runInAction } from 'mobx'
+
+import { t } from '@i18n'
+import dirinfoStore from '@store/dirinfo'
+import { TIcons } from '@ui/icon/icons'
+import datasetProvider from '@service-providers/dataset-level/dataset.provider'
+import { showToast } from '@utils/notifications/showToast'
+
+export class DescriptionCardStore {
+  public activeDtree: string = ''
+
+  private _dsName = ''
+  private _descriptions: { [key: string]: string } = {}
```

Comment:
is this a cache or something? Why would you need it, you don't make any request to get the description specifically anyway, so why waste memory for it?